02.calendar/calendar.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+def calendar
+
+  require 'date'
+  require 'optparse'
+
+  #monthとyearの設定
+  month = Date.today.month
+  year = Date.today.year
+
+  opt = OptionParser.new
+  opt.on('-m [VAL]') { |v| month = v }
+  opt.on('-y [VAL]') { |v| year = v }
+  opt.parse!(ARGV)
+  month = month.to_i
+  year = year.to_i
+
+
+  #表示内容を生成
+  puts "       #{month}月 #{year}"
+  puts " 日 月 火 水 木 金 土"
+  firstDay = Date.new(year, month, 1)
+  lastDay = Date.new(year, month, -1)
+  dayOfWeek = firstDay.wday
+  dayOfWeek.times{print "   "}
```

Comment:
```suggestion
  print "   " * dayOfWeek
```

Rubyだと`*`で文字列を繰り返すことができます